### ISSUE TITLE
Dev - Layout issues

### ISSUE THREAD

**Issue by leonieherma1** [hypha-dao/dho-web-client#390]
- [x] Copy "Yes", "No", "Abstain" on button should be in the center
<img width="322" alt="Screenshot 2021-05-03 at 10 22 46" src="https://user-images.githubusercontent.com/75991832/116903447-a77b1580-abf9-11eb-997d-f8cbd56c72e3.png">

- [x] Wallet is empty view
- [x] Picker next to commitment is not aligned: https://cdn.discordapp.com/attachments/783371933127409674/838818464763478076/unknown.png
- [x] Avatar is empty
- [ ] "Checkbox Icon" doesn't show up in sidebar after voting (Julio & Augsuto Chrome)
https://cdn.discordapp.com/attachments/783371933127409674/838821432648794152/unknown.png
- [x] Edit profile (bio) should update automatically without refreshing
- [x] Profile error not centered
- [x] Fix icon on lunear cycles
- [x] Edit icon to change avatar
https://cdn.discordapp.com/attachments/722555929526140969/838814557223911455/unknown.png

- [ ] Propose role assignment (e.g. 55 periods Julio) allowed to save but when the user proposes he gets an error message: "assertion failure with message: period_count must be less than 26. You submitted: 55" - Validation should happen before saving

- [ ] When I try to change my profile photo (and it is too big) I should get an error message. It just doesn't update.

**Comment by lukegravitt** [comment]
These 2 issues should each have a separate issue created. They are too involved to do for the release.

"Checkbox Icon" doesn't show up in sidebar after voting (Julio & Augsuto Chrome)
https://cdn.discordapp.com/attachments/783371933127409674/838821432648794152/unknown.png

Propose role assignment (e.g. 55 periods Julio) allowed to save but when the user proposes he gets an error message: "assertion failure with message: period_count must be less than 26. You submitted: 55" - Validation should happen before saving

**Comment by lukegravitt** [comment]
For this one, I just removed the buttons (they were particularly ugly on firefox)
 
Picker next to commitment is not aligned: https://cdn.discordapp.com/attachments/783371933127409674/838818464763478076/unknown.png
